main/editor.js

```diff
@@ -19,11 +19,27 @@ let editor = null;
 
 const openEditorWindow = () => {
```

Comment:
Should this take src as the filepath to the video that should be edited? @karaggeorge

Reply:
And to pass that to the renderer, should i use the IPC stuff and listen for that event in the component, or should we use a query param like `?src=` when opening the HTML file?